c3pzero_description/urdf/c3pzero_base.xacro

```diff
@@ -275,8 +275,8 @@
             </vertical>
           </scan>
           <range>
-            <min>0.120000</min>
-            <max>3.5</max>
+            <min>0.05</min>
+            <max>20</max>
```

Comment:
I mean, sure, but the lidar we're using isn't a 20 meter lidar

Reply:
Do you have a datasheet or model name, I can look up to?

Reply:
https://emanual.robotis.com/docs/en/platform/turtlebot3/appendix_lds_01/

Reply:
Oh lol seems the prior parameters were correct, from https://emanual.robotis.com/assets/docs/LDS_Basic_Specification.pdf

> Detection distance 120mm ~ 3,500mm

Then the mapping problem is just related with range.

Reply:
maybe, but costmaps should still clear. also, the videos on that page imply we should be able to get better performance

Reply:
You know though, I also might be conflating some things in my head

Reply:
Changed back to correct paramters, appreciate if you can take a test.

Reply:
yup. gonna have to wait till later in the day now though, sorry

Reply:
So to my understanding currently if Laser is out of max range (inf values) it just doesn't say it's empty / free space instead just leaves it unknown. I don't know if there is a parameter to change this behaviour.